src/lib/item-types.js

```diff
@@ -130,11 +130,6 @@ const itemTypes = {
       }
     }
   },
-  'experiment': {
-    name: 'experiment', // TODO Redundant, change this structure
-    restCollectionName: 'experiments',
-    primaryKey: 'urn'
-  },
```

Comment:
We will still need this definition, since it is used for the experiment creator, editor, and viewer.

Reply:
Yes, you're correct.